features/networking/egress-ingress.feature

```diff
@@ -300,3 +300,50 @@ Feature: Egress-ingress related networking scenarios
     When I execute on the pod:
       | curl | --head | www.test.com |
     Then the step should succeed
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-19615
+  @admin
+  Scenario: Iptables should be updated with correct endpoints when egress DNS policy was used
+    Given I have a project
+    When I run oc create over "<%= BushSlicer::HOME %>/testdata/networking/list_for_pods.json" replacing paths:
+      | ["items"][0]["spec"]["replicas"] | 1 |
+    Then the step should succeed
+    And 1 pods become ready with labels:
+      | name=test-pods |
+    Given I use the "test-service" service
+    And evaluation of `service.ip(user: user)` is stored in the :service_ip clipboard
```

Comment:
I was going to suggest `service.ip_url()` for future IPv6 support, but we only have `ip_url` method for Pod,
but we do have `service.url()` method, should we be using that instead of hard-coding port `27017` below?

Reply:
see also https://github.com/openshift/verification-tests/pull/1210/files

Reply:
Above two line can be shorted as
```
And evaluation of `service("test-service").ip` is stored in the :service_ip clipboard`
```

Reply:
#1210 was merged so we can do

```
And evaluation of `service("test-service").url` is stored in the :service_url clipboard`

...

| /usr/bin/curl | -k | <%= cb.service_url %> |

```

Reply:
Thanks @rbbratta, updated it.